bazel/external/jinja.BUILD

```diff
@@ -1,8 +1,28 @@
 licenses(["notice"])  # Apache 2
 
+# Starting with 2.11, the jinja2 distribution does not keep the source files in a directory with
+# the expected package name (it uses 'src/jinja2' instead of 'jinja2'), so we provide a genrule to
+# copy src/jinja2/*.py to jinja2/*.py.
+
+src_files = glob(["src/jinja2/*.py"])
+
+genrule(
+    name = "jinja_files",
+    srcs = src_files,
+    outs = [f.replace("src/jinja2/", "jinja2/") for f in src_files],
+    cmd = "\n".join(
+        ["mkdir -p $$(dirname $(location %s)) && cp $(location %s) $(location :%s)" % (
+            f,
+            f,
+            f.replace("src/jinja2/", "jinja2/"),
+        ) for f in src_files],
+    ),
+    visibility = ["//visibility:private"],
+)
```

Comment:
As discussed on Slack, I reckon https://github.com/bazelbuild/rules_python would be a better option and probably more maintainable. I don't think it would be a ton of work to get happening, but famous last words. If you prefer you can file an issue and put a TODO here.

Out of curiosity, would https://docs.bazel.build/versions/master/be/python.html#py_binary.imports have been any help?

Reply:
I couldn't get imports to work, but I've had only limited exposure to bazel.

Reply:
Sure; I'm actually doing some of the required plumbing here in https://github.com/envoyproxy/envoy/pull/11058, so in the near future we might have `pip3` support in the repository.